content/en/tracing/llm_observability/quickstart.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Quickstart
+---
+<div class="alert alert-info">LLM Observability is in public beta.</a></div>
+
+Our quickstart docs make use of our Python SDK. For detailed usage, see [the SDK documentation][1]. If your application is written in another language, you can create traces by calling the [API] instead.
+
+## Jupyter notebook quickstart
+
+To try out LLM Observability in a series of guided notebooks, follow the instructions in [the LLM Observability onboarding repo][2].
+
+## Command line quickstart
+
+Use the steps below to run a simple Python script that generates an LLM Observability trace.
+
+### Prerequisites
+
+- LLM Observability requires a Datadog API key (see [the instructions for creating an API key][7]).
+- The example script below uses OpenAI, but you can modify it to use a different provider. To run the script as written, you need:
+    - An OpenAI API key stored in your environment as `OPENAI_API_KEY`. To create one, see [Account Setup][4] and [Set up your API key][6] in the OpenAI documentation.
+    - The OpenAI Python library installed. See [Setting up Python][5] in the OpenAI documentation for instructions.
+
+### 1. Install the SDK
+
+Install the following `ddtrace` package hash:
+
+{{< code-block lang="shell" >}}
+pip install git+https://github.com/DataDog/dd-trace-py.git@c03cc8764c508cf53ea38b7b0f0eb6cb39627d91
+{{< /code-block >}}
+
+### 2. Create the script
+
+The Python script below makes a single OpenAI call. Save it as `quickstart.py`.
+
+{{< code-block lang="python" filename="quickstart.py" >}}
+import os
+from openai import OpenAI
+
+oai_client = OpenAI(api_key=os.environ.get("OPENAI_API_KEY"))
+
+completion = oai_client.chat.completions.create(
+    model="gpt-3.5-turbo",
+    messages=[
+        {"role": "system", "content": "You are a helpful customer assistant for a furniture store."},
+        {"role": "user", "content": "I'd like to buy a chair for my living room."},
+    ],
+)
+{{< /code-block >}}
+
+### 3. Run the script
+
+To generate a trace, run the Python script with the following shell command:
```

Comment:
Nit; "Run the following python script and a trace of the openai call will be sent to datadog" or something similar, to emphasize the trace is being _sent_ to datadog

Reply:
Since the idea of generating a trace in it itself might be kinda vague